### ISSUE TITLE
Reminder emails

### ISSUE THREAD

**Issue by ashbowe** [nmrf/sadata#12]
Hi Timo,

What did we agree on the process for reminder emails in the end? Will there be a link for the user to click on or do they need to sign in themselves and then find the relevant indicator?

Thanks,

Ash

**Comment by tmfrnz** [comment]
Yes, reminder emails will have a link to the relevant indicator (still on our TODO list https://github.com/impactoss/impactoss-server/issues/232)

**Comment by ashbowe** [comment]
Great, thanks.
